src/Compatibility/Core/src/AppHostBuilderExtensions.Android.cs

```diff
@@ -22,14 +22,18 @@ static void OnConfigureLifeCycle(IAndroidLifecycleBuilder android)
 					var services = MauiApplication.Current.Services;
 					var mauiContext = new MauiContext(services, app);
 					var state = new ActivationState(mauiContext);
+#pragma warning disable CS0612 // Type or member is obsolete
 					Forms.Init(state, new InitializationOptions { Flags = InitializationFlags.SkipRenderers });
+#pragma warning restore CS0612 // Type or member is obsolete
```

Comment:
In order for us to still use the Forms.Init in the builder, we can just do this. Eventually, all this goes away and it is pretty slim right now anyway.